### PR TITLE
Change maven repository order to use google first

### PR DESCRIPTION
**Pull request by iOrcohen**
#### Description of changes

In the past day, jCenter had troubles in their Maven repository. That makes the android ```amplify-js:pushnotification``` package impossible to build, because there is a google-services dependency from jCenter which causes a timeout during builds that use the ```amplify-js:pushnotification``` package in the react native app.

image

In this PR I change the order of the maven repository to make google maven repository first, and solve the issue.

#### Description of how you validated changes
I run a build for the pushnotification package, which succeded. 
Also I make this change inside my react-native app and it worked.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/pushnotification/android/build.gradle [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 buildscript {
     repositories {
-        jcenter()
-				google()
+        google()
+        jcenter()
```
Comment (cshfang) [inline packages/pushnotification/android/build.gradle:6]:
```suggestion
        mavenCentral()
```

File: packages/pushnotification/android/build.gradle [hunk 2]
```diff
@@ -13,10 +13,8 @@ buildscript {
 
 allprojects {
     repositories {
+        google()
         jcenter()
```
Comment (cshfang) [inline packages/pushnotification/android/build.gradle:17]:
```suggestion
        mavenCentral()
```

**Review by cshfang — COMMENTED** [review]
Hey @iOrcohen, thanks for submitting this PR. The change looks fine to me but I'm wondering if you agree we might as well just deprecate jcenter and migrate to mavenCentral.

**Comment by tannerabread** [comment]
Closing this due to inactivity. This should be handled by the upcoming versions of push notification

**Pull request closed without merging**